node/src/BaseClient.ts

```diff
@@ -1056,6 +1057,30 @@ export class BaseClient {
         return this.createWritePromise(createLLen(key));
     }
 
+    /**
+     * Sets the list element at `index` to `element`.
+     * The index is zero-based, so 0 means the first element, 1 the second element and so on.
```

Comment:
```suggestion
     * The index is zero-based, so `0` means the first element, `1` the second element and so on.
```
You need to wrap numbers with backticks (`` ` ``) everywhtere for better visibility.